### ISSUE TITLE
whatshap stats should ignore non-variants in gVCF files

### ISSUE THREAD

**Issue by marcelm** [whatshap/whatshap#359]
gVCF files represent even non-variant sites as records. We should just ignore all non-variant sites in the input VCF so that the result is identical to the one that one would have gotten had one run the variant caller in non-gVCF mode.

This could be considered backwards incompatible and way warrant a major version bump.

See #358.

**Comment by PlatonB** [comment]
@marcelm Non-variant sites are very important for the creation of platinum genomes. Please do not ignore them. It would be nice if `0/0` would simply be converted to `0|0`.

**Comment by marcelm** [comment]
@PlatonB This issue is about gVCFs as input to `whatshap stats`. Can you please add your suggestion to #502? We’ll use that to track gVCF support in `whatshap phase`.
